consensus/istanbul/backend/announce.go

```diff
@@ -159,19 +92,28 @@ func (sb *Backend) sendAnnounceMsgs() {
 	}
 }
 
-func (sb *Backend) generateIstAnnounce() ([]byte, error) {
+func (sb *Backend) generateIstAnnounce() (*istanbul.Message, error) {
 	block := sb.currentBlock()
-	selfEnode := sb.Enode()
 
-	if selfEnode == nil {
-		sb.logger.Error("Enode is nil in sendIstAnnounce")
-		return nil, nil
-	}
+	var enodeUrl string
+	if sb.config.Proxied {
+		if sb.sentryNode != nil {
+			enodeUrl = sb.sentryNode.externalNode.String()
+		} else {
+			sb.logger.Debug("Proxied node has no attached sentries")
```

Comment:
It is possible for a validator operator to start up a proxied validator and then start up a sentry afterwards.

Between the startups, this condition can happen, and I felt it was not strictly an `error`.  So I went with `debug`.  However, I am not that wedded to it.

Reply:
Slight preference for ERROR, since I would consider that state (proxied validator running but not connected to a sentry) to be something that needs to be addressed